tfjs-backend-webgpu/src/matmul_packed_webgpu.ts

```diff
@@ -21,26 +21,23 @@ import {getMainHeaderString as main, WebGPUProgram} from './webgpu_program';
 import {computeDispatch, computeWorkgroupInfoForMatMul} from './webgpu_util';
 
 export function matMulReadFnSource(
-    batchAEqualOne: boolean, batchBEqualOne: boolean, transposeA: boolean,
-    transposeB: boolean, fitAOuter = false, fitBOuter = false, fitInner = false,
-    component = 1) {
+    transposeA: boolean, transposeB: boolean, fitAOuter = false,
+    fitBOuter = false, fitInner = false, component = 1) {
   util.assert(
       transposeA && component === 1 || !transposeA,
       () => `transposeA ${transposeA} is not compatible with component size ${
           component}`);
   const sampleA = `
-      let batch = ${batchAEqualOne ? '0' : 'batchIn'};
       ${
-      transposeA ? `value = getA(batch, col, row);` :
-                   `value = getA(batch, row, col);`}
+      transposeA ? `value = getA(batchIn, col, row);` :
```

Comment:
nit: Since we don't need to change `batchIn` anymore, please rename `batchIn` to `batch`.